pkg/lite/lite.go

```diff
@@ -174,7 +175,7 @@ func start(ctx context.Context, opts StartOpts) error {
 
 	queueOpts := []redis_state.QueueOpt{
 		redis_state.WithIdempotencyTTL(time.Hour),
-		redis_state.WithNumWorkers(100),
```

Comment:
A bit of a nit, but is it acceptable that users can break their instance by providing `--queue-workers 0`?